bolt/include/bolt/Core/DebugNames.h

```diff
@@ -0,0 +1,192 @@
+//===- bolt/Core/DebugNames.cpp - Debugging information handling ---*- C++
```

Comment:
Do you want to write "debug name suppport" instead of "Debugging information handling"